### PR TITLE
Add 3GPP TS 29.061 attributes to NASREQ Diameter provider

### PR DESCRIPTION
**Pull request by surik**

### CODE REVIEW

**Comment by surik** [comment]
I marked it as WIP because it may have conflicts with `feature/pcc-rules` branch. Some things will have to be polished after merge of that feature branch, like: using common helpers, rework tests and diameter messages as lists instead of records.

**Comment by tlnd** [comment]
@RoadRunnr, @surik  could you please co-operate on getting this merged. Maybe some of the commits in `feature/pcc-rules` should go to master as well, maybe others not. Thanks

**Comment by RoadRunnr** [comment]
----- On Aug 7, 2017, at 1:40 PM, Timo Lindhorst <notifications@github.com> wrote: 

> [ https://github.com/roadrunnr | @RoadRunnr ] , [ https://github.com/surik |
> @surik ] could you please co-operate on getting this merged. Maybe some of the
> commits in feature/pcc-rules should go to master as well, maybe others not.
> Thanks

feature/pcc-rules is very much work in progress. I have another stack of changes on top of what's already there that I'm working on. 
There is IMHO no point in merging anything from the two branches as long as not both are at least somewhat stable.

**Comment by RoadRunnr** [comment]
here is the plan, I'm going to move everything that add diameter dictionaries into a new PR. Once that is merge, @surik can rebase this PR onto that (minus the dictionary in this PR).

Once the PCC work is merged, we can try to combine and reuse as much code as possible (PCC work does not touch the code in this PR).

File: dia/diameter_nasreq_rfc7155.dia [hunk 1]
```diff
@@ -171,6 +172,32 @@
             * [ Tunneling ]
             * [ Proxy-Info ]
             * [ Route-Record ]
+              [ 3GPP-IMSI]
```
Comment (RoadRunnr) [inline dia/diameter_nasreq_rfc7155.dia]: No, please use diameter_3gpp_ts29_061_sgi instead.

**Pull request merged**